subsys/net/ip/icmp.c

```diff
@@ -0,0 +1,390 @@
+/** @file
+ * @brief ICMP related functions
+ */
+
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Use highest log level if both IPv4 and IPv6 are defined */
+#if defined(CONFIG_NET_IPV4)
+#define MY_LOG_LEVEL CONFIG_NET_ICMPV4_LOG_LEVEL
+#endif
+
+#if defined(CONFIG_NET_IPV6)
+#if defined(MY_LOG_LEVEL)
+#if MY_LOG_LEVEL < CONFIG_NET_ICMPV6_LOG_LEVEL
+#define MY_LOG_LEVEL CONFIG_NET_ICMPV6_LOG_LEVEL
```

Comment:
Shouldn't `MY_LOG_LEVEL` be undefined before? Looks like symbol redefinition right now.